data/json/monsters/turrets.json

```diff
@@ -203,7 +207,7 @@
         "ammo_type": "762_51",
         "fake_skills": [ [ "gun", 8 ], [ "rifle", 8 ] ],
         "fake_dex": 12,
-        "ranges": [ [ 0, 30, "DEFAULT" ] ],
+        "ranges": [ [ 0, 60, "DEFAULT" ] ],
```

Comment:
Also, I'm not sure about extending the range to the full limit of the reality bubble, it's very punishing.

Reply:
Although max firing range is 60, their vision range is 50 at best, so they will start firing only at 50 or less.

Reply:
Oh, missed that.